test/specs/print_utils/tests/01.print_utils.js

```diff
@@ -132,5 +133,13 @@ exports.execute = function (options) {
             }
         });
 
+        it('module._renderTemplate() should function correctly', function() {
+            expect(module._renderTemplate("My name is {{name}}", {name: 'John'})).toBe("My name is John");
+            expect(module._renderTemplate("My name is {{name}}", { name: null })).toBe(null);
```

Comment:
@robes I will add that testcase. Though the output for above template will be `My name is ` instead of null. This is because the object that we are passing is empty. Mustache replaces null values with `''` (empty string). So it won't return null.

If you want then I can handle this case, to check all properties in the template for their existence in the object and null value.